### PR TITLE
Updated developers list and enabled CD | plugin-simplify-qa-connector

### PR DESCRIPTION
**Pull request by Simplify3x**
# Link to GitHub repository

https://github.com/jenkinsci/simplify-qa-connector-plugin

# When modifying release permission

If you are modifying the release permission of your plugin or component, fill out the following checklist:

```[tasklist]
### Release permission checklist (for submitters)
- [x] The usernames of the users added to the "developers" section in the .yml file are the same the users use to log in to [accounts.jenkins.io](https://accounts.jenkins.io/).
- [x] All users added have logged in to [Artifactory](https://repo.jenkins-ci.org/) and [Jira](https://issues.jenkins.io/) once.
- [ ] I have mentioned an [existing team member](https://github.com/orgs/jenkinsci/teams) of the plugin or component team to approve this request.
```

## When enabling automated releases (cd: true)

Follow the [documentation](https://www.jenkins.io/doc/developer/publishing/releasing-cd/) to ensure, your pull request is set up properly. Don't merge it yet.  
In case of changes requested by the hosting team, an open PR facilitates future reviews, without derailing work across multiple PRs.

### Link to the PR enabling CD in your plugin

https://github.com/jenkinsci/simplify-qa-connector-plugin/pull/9

```[tasklist]
### CD checklist (for submitters)
- [x] I have provided a link to the pull request in my plugin, which enables CD according to the documentation.
```

```[tasklist]
### Reviewer checklist
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.
```
There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it.

### CODE REVIEW

**Comment by Simplify3x** [comment]
Can we get an update on this?

**Review by NotMyFault — CHANGES_REQUESTED** [review]
Screenshot 2023-12-11 at 12 58 24

The link should be the link to your plugin's PR, not the RPU PR.
Changing the path implies a GAV change, which is non-backwards compatible. Please revert the change and stick to the pre-existing path.

**Comment by Simplify3x** [comment]
@NotMyFault can you provide some clarification or maybe a link, as to which two branches need to be merged in the Plugin's PR?

**Comment by NotMyFault** [comment]
> NotMyFault can you provide some clarification or maybe a link, as to which two branches need to be merged in the Plugin's PR?

Which part is unclear? You prepare the changes outlined in the documentation, raise a PR including all plugin modifications, and we review them. If everything is right, this PR will be merged.

**Comment by Simplify3x** [comment]
The changes outlined in the documentation are already a part of the latest commit in the master branch of the repository.
There is no requirement to raise a PR.
You are good to review the code changes.

**Comment by NotMyFault** [comment]
> The changes outlined in the documentation are already a part of the latest commit in the master branch of the repository.
There is no requirement to raise a PR.

That exactly is the requirement, allowing us to review them _before_ integrating into the default branch. Because, as in your case, the changes made are incomplete and cannot be reviewed in a PR-like manner.

**Comment by Simplify3x** [comment]
Hi @NotMyFault,
We have made the changes and created a new PR as per mentioned in the documentation.
Kindly review the changes and provide your valuable feedback.

**Comment by NotMyFault** [comment]
Looks good so far, please rebase your PR with the latest changes on the default branch.

**Comment by Simplify3x** [comment]
Rebase Done

**Comment by NotMyFault** [comment]
> Rebase Done

Screenshot 2023-12-18 at 21 19 03

This PR has not been rebased.

**Comment by Simplify3x** [comment]
There was no `update with rebase` option so we merged it instead.
Trying to revert and re-create a new PR.

**Comment by Simplify3x** [comment]
Can we get some update on this?

**Comment by alecharp** [comment]
@Simplify3x please, on your local clone of this repository (`repository-permissions-updater`) the following commands:

```
git pull -a
git rebase upstream/master
```

if the remote towards https://github.com/jenkins-infra/repository-permissions-updater.git is not called `upstream` on your local clone, please adapt the command accordingly.

**Comment by Simplify3x** [comment]
> @Simplify3x please, on your local clone of this repository (`repository-permissions-updater`) the following commands:
> 
> ```
> git pull -a
> git rebase upstream/master
> ```
> 
> if the remote towards https://github.com/jenkins-infra/repository-permissions-updater.git is not called `upstream` on your local clone, please adapt the command accordingly.

image

This account does not have the write access to the RPU repo

**Comment by alecharp** [comment]
@Simplify3x when you open a pull request against a repository (`jenkins-infra/repository-permissions-updater`), you link a branch from your fork and the pull request. Anything that you push to the branch will be seen in the pull request. So, when you merged the commits from `jenkins-infra:master` into your `master` branch, and push that commit into your fork, it's showed here (https://github.com/jenkins-infra/repository-permissions-updater/pull/3685/commits/e0a0408ca91eaa31d142da72519f8196e2829224). 

> Now, to rebase and merge these changes to the master branch of [RPU](https://github.com/jenkins-infra/repository-permissions-updater.git), it is only possible by you or your team.
[We](https://github.com/Simplify3x) do not have push access to the master branch of [RPU](https://github.com/jenkins-infra/repository-permissions-updater.git)

You don't need write access on the `jenkins-infra` repository to do that. That's why we asked you do to the rebase.

File: permissions/plugin-simplify-qa-connector.yml [hunk 1]
```diff
@@ -4,6 +4,11 @@ github: "jenkinsci/simplify-qa-connector-plugin"
 issues:
   - jira: '28523'  # simplify-qa-connector-plugin
 paths:
-  - "io/jenkins/plugins/simplify-qa-connector"
+  - "io/jenkins/plugins/OldUtil"
+  - "io/jenkins/plugins/SQA"
```
Comment (alecharp) [inline permissions/plugin-simplify-qa-connector.yml:8]: I'm not sure this is something you want to include in your pull request.

**Review by alecharp — CHANGES_REQUESTED** [review]

File: permissions/plugin-simplify-qa-connector.yml (hunk 1 above)
Comment (Simplify3x) [reply permissions/plugin-simplify-qa-connector.yml:8]: Sure, will include that.

Comment (NotMyFault) [reply permissions/plugin-simplify-qa-connector.yml:8]: > We want to include that.

The response lacks a sufficient justification of what the path is supposed to represent in terms of enabling CD.

Comment (Simplify3x) [reply permissions/plugin-simplify-qa-connector.yml:8]: What kind of justification do you require?

Comment (NotMyFault) [reply permissions/plugin-simplify-qa-connector.yml:8]: > What kind of justification do you require?

The change proposed lacks a rationale reasoning why you want to modify your release paths, when the PR description and title claim to enable CD only.

Additionally, the repository isn't set up to support project inheritance (i.e., no reactor, no module aggregation, etc.). This leads to the fact that you can't release multiple artifacts under multiple artifact IDs, even if you want to do that.
It's worth mentioning that this is a complex procedure, you commonly don't answer with "We want to include that.", and therefore I agree with alecharp that you should remove this from the PR.

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**